ninja-core/src/test/java/ninja/utils/CookieDataCodecTest.java

```diff
@@ -184,4 +191,32 @@ public void decode_values_with_gibberish_in_them() throws UnsupportedEncodingExc
         decode(outMap, data);
         assertEquals(0, outMap.size());
     }
+
+    @Rule public ExpectedException thrown = ExpectedException.none();
+    /* testedClasses: CookieDataCodec */
+    /*
+     * Test generated by Diffblue Deeptest.
+     * This test case covers the entire method.
+     */
+    @Test
+    public void constructorOutputVoid() {
+      // Act, creating object to test constructor
+      CookieDataCodec objectUnderTest = new CookieDataCodec();
+      // Method returns void, testing that no exception is thrown
+    }
+    /*
+     * Test generated by Diffblue Deeptest.
+     * This test case covers:
+     *  - conditional line 77 branch to line 78
+     */
+    @Test
+    public void safeEqualsInputNotNullNotNullOutputFalse() {
```

Comment:
Is this the only test?

Reply:
Yes. I added this test because it covers the only line in the class that hasn't already been covered. I thought it was a nice touch!

Reply:
Exceptional 👍

Reply:
👍

Reply:
:+1: